setup.md

```diff
@@ -0,0 +1,72 @@
+# Setting up your local development environment for PyGMT
+
+We will use Git, Anaconda, and the `conda` package manager to install and
+manage the development environment.
+
+This guide assumes a basic understanding of git using the command line. For
+more help, attend the optional setup session on August 17 and/or review the
+[Software Carpentry](https://software-carpentry.org/)'s courses on
+[the Unix Shell](https://swcarpentry.github.io/shell-novice/) and
```

Comment:
More as a general question, are we assuming that users are on UNIX-type shells (i.e. macOS/Linux) instead of Windows?

Reply:
I could add reference to git for windows to this introductory paragraph for those on Windows. I would prefer to teach while assuming access to a bash emulator, but we should make everything general enough to work on macOS/Windows/Linux.

Reply:
Ok, just something to keep in mind. I see a `cd` command in the setup in the setup instructions (which should be cross-platform), and the `git` commands should work ok too. But things will get tricky once we try things like `make install` so we need to be careful.

Reply:
Thanks, good point. I've added a suggestion to use Git for Windows in https://github.com/GenericMappingTools/2021-eswn-pygmt-workshop/pull/2/commits/e5a944cbccd169c089fe8b370d1e8898598a175b and removed the Unix Shell course link since it's not as necessary. That is what I use when working on Windows, so it's what I can mentor most effectively for though there are other options.